src/commonTest/kotlin/fr/acinq/lightning/channel/InteractiveTxTestsCommon.kt

```diff
@@ -1080,18 +1084,22 @@ class InteractiveTxTestsCommon : LightningTestSuite() {
             lockTime: Long
         ): Either<FundingContributionFailure, FundingContributions> {
             val channelId = randomBytes32()
+            val fundingTxIndex = 0L
```

Comment:
Shouldn't this be an argument of `createSpliceContributions`?

Reply:
Done in 4b3df591.